### PR TITLE
npm-prerelease.yml automated workflow

### PR DESCRIPTION
**Pull request by shefalijoshi**
Workflow to automatically publish an npmjs package when a Github pre-release is created

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

### CODE REVIEW

**Comment by akhenry** [comment]
@shefalijoshi  Looks like some comments to resolve before we can merge this?

File: npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# This workflow will run tests using node and then publish a package to npmjs when a prerelease is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
```
[the remark this answers is not in the document]
Comment (shefalijoshi) [reply npm-publish.yml:6]: Yes, I suspect so.

File: npm-publish.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish --access public --tag unstable
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```
[the remark this answers is not in the document]
Comment (shefalijoshi) [reply npm-publish.yml:33]: Done.

**Review by unlikelyzero — APPROVED** [review]
LGTM

**Pull request merged**